pages/test.js

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect, useState } from 'react';
+import { Button } from 'components/ui/button';
+import { useLocalStorage } from 'hooks/hooks';
+import { issueRevokableCredential } from 'utils/issue-crendentials';
+import { createCreditScoreCredential } from '_credentials/equinet';
+import { getRegistry, revokeCredential } from 'utils/dock-registries';
+import { toast } from 'sonner';
+import userStore from 'store/appStore';
+
+export default function Test() {
+  const [revokableCredential, setRevokableCredential] = useLocalStorage('revokableCredential', '');
```

Comment:
I saw `useLocalStorage('revokableCredential', '');` a few times in this PR and I want to ask why don't we also use a store for it?

Reply:
because it should be in cache. When the app reloads will keep this record for the Equinet page, revoke a crendential using the id. We need to store it in cache instead.

Reply:
And the values from `https://github.com/docknetwork/dock-bank-demo/blob/dev/store/appStore.js` shouldn't? I don't know if after the page reloads you need `did` or `email` but I want to make sure that part is not overlooked.